### PR TITLE
Update SolDefines.py

### PR DESCRIPTION
**Pull request by 5ource**

### CODE REVIEW

File: SolDefines.py [hunk 1]
```diff
@@ -288,4 +288,22 @@ def solStructure(type_id):
         'structure':    '>III',
         'fields':       ['sol_version', 'solmanager_version', 'sdk_version'],
     },
+    {
+        'type':         SENS_MB7363_D2S2N1L1G1,
```
Comment (twatteyne) [inline SolDefines.py]: please update:
- change names from `SENS_MB7363_D2S2N1L1G1` to `SOL_TYPE_SENS_MB7363_D2S2N1L1G1`
- add these values to the list of `SOL_TYPE_` values

**Comment by keomabrun** [comment]
@5ource Please also change the SOL_TYPE_ names in the registry.md file and add the fields description at the end of the file too.

**Comment by keomabrun** [comment]
@5ource why did you close it ?

**Comment by 5ource** [comment]
@keomabrun I thought i did something wrong by creating branches called "5ource-patch-1.." so I deleted them. I wanted to commit to the master. So I should create a branch and work on it regardless of master?

**Comment by keomabrun** [comment]
Usually, we work either directly on branch develop or develop_XX (where XX is the issue number). Then we merge develop to master when we publish a new release.

I reused some of the code from the branch you deleted and will push it soon.

**Comment by 5ource** [comment]
Hi Keoma,

I created a branch from develop (called feather) and it should be the most
up to date correct one.

On Fri, Jun 17, 2016 at 12:12 AM, Keoma Brun notifications@github.com
wrote:

> Usually, we work either directly on branch develop or develop_XX (where XX
> is the issue number). Then we merge develop to master when we publish a new
> release.
> 
> I reused some of the code from the branch you deleted and will push it
> soon.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/realms-team/sol/pull/37#issuecomment-226697265, or mute
> the thread
> https://github.com/notifications/unsubscribe/AI6g7fyapWO_wx4VBW1kVxDrZtkYDqwWks5qMkjDgaJpZM4IpZzk
> .

## 

Sami Andrew Malek
PhD Student, Department of Civil and Environmental Engineering
621 Sutardja Dai Hall (CITRIS Building)
UC Berkeley, CA 94720-1758
sami.malek@berkeley.edu

**Pull request closed without merging**